functions/ap/users/[id]/outbox.ts

```diff
@@ -17,6 +13,7 @@ const headers = {
 	'content-type': 'application/json; charset=utf-8',
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars -- TODO: use userKEK
```

Comment:
@xtuc here and in some other places where I was unsure if the variables are needed or not I left the variables and added these TODO eslint disabling comments, let me know if that's ok (or if we can get rid of the variables)